website/profile/views.py

```diff
@@ -57,15 +56,14 @@ def get_public_projects(uid=None, user=None):
 def get_public_components(uid=None, user=None):
     user = user or User.load(uid)
     # TODO: This should use User.visible_contributor_to?
-
+    # TODO: Should this be limited for users with many projects / components?
```

Comment:
And/or does it has to be cast to a list? Get public projects isn't being cast to a list.